swri_transform_util/include/swri_transform_util/local_xy_util.h

```diff
@@ -80,14 +80,25 @@ namespace swri_transform_util
       double& longitude);
 
   /**
-   * Utility class for converting between WGS84 lat/lon and an ortho-rectified
+   * @brief Utility class for converting between WGS84 lat/lon and an ortho-rectified
```

Comment:
Not that it matters much, but I think the @brief tag here and elsewhere is redundant.  The first paragraph is automatically considered the brief description by Doxygen  and I don't think adding the tag adds anything for human readability.